content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,281 @@ Feature: Is it Friday yet?
 
 Congratulations! You've got your first green Cucumber scenario.
 
+# Evaluate a scenario
+Now that we've forced a scenario to pass, let's write one that passes or fails based on logic. Let's update our statement to evaluate whether or not `today` is equal to `"Friday"`.
+
+{{% block "javascript" %}}
+```javascript
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```ruby
+def is_it_friday?(day)
+  answer = (day == 'Friday') ? 'TGIF' : 'Nope'
```

Comment:
Don't need local assigns. Probably prefer if/else over ternaries.

In the code below make the same changes

Reply:
Totally forgot Ruby returns the last statement by default, I'll remove the assignment. I think that ternary is preferred for this since it is [extremely simple](https://github.com/airbnb/ruby#ternary-operator)?

Reply:
Can do I guess. I'm against them though. But I'm not gonna hold things up since it's only a guide. Not an enforcement.

Reply:
Add Java:

{{% block "java" %}}
```java
    static String isItFriday(String today) {
        if (today.equals("Friday")) {
            return "TGIF";
        }
        return "Nope";
    }
```
{{% /block %}}

Reply:
And again, mark the source code as java ;)

Reply:
K - so I think the Ruby bit should look like this then (going into my next commit). Also, we want to take off the ? since it is not a [predicate method](https://github.com/airbnb/ruby#predicate-method-names) right?
```ruby
def is_it_friday(day)
  if day == 'Friday'
    'TGIF'
  else
    'Nope'
  end
end
```

Reply:
Yes, I'd prefer if/else (but can't comment on Ruby syntax)

Reply:
Correct @mikegfisher - Arguably it should be but for your example just dropping the `?` is fine